zerver/views/video_calls.py

```diff
@@ -259,6 +259,7 @@ def join_bigbluebutton(
                 "meetingID": meeting_id,
                 "password": password,
                 "fullName": request.user.full_name,
+                "createTime": payload.find("createTime").text,
```

Comment:
Can you add a comment explaining how this parameter works / its purpose?  It seems a little surprising that we'd send `createTime` in this call.

Reply:
Will add the comment tomorrow.
The BigBlueButton documentation says:
```
Third-party apps using the API can now pass createTime parameter (which was created in the create call), BigBlueButton will ensure it matches the ‘createTime’ for the session. If they differ, BigBlueButton will not proceed with the join request. This prevents a user from reusing their join URL for a subsequent session with the same meetingID.
```

The createTime is (as far as I know, I have tested) provided by the "create" request before, even if the meeting is already created.
This is used so that a user which previously had access to a BigBlueButton "join this video call link" cant save the URL he is redirected to (before joining the meeting) and use it again when the same link is "join this video call link" is used.


This wont break any old links. But the meeting name and signing changes will.